src/modules/popmotion-controller/popmotion-controller.ts

```diff
@@ -244,8 +293,27 @@ export const popmotionController = (config: PopmotionControllerConfig = {}): Run
             { x: e.touches[1].clientX, y: e.touches[1].clientY }
           );
           isMulti = true;
+          setDebugBorder('1px solid blue');
         }
+
         if (state.isPressing && e.touches.length === 1) {
+          if (enablePanOnWait) {
+            // if there is a delay between the touch-start and the 1st touch-move of < xms, then treat that as a PAN, 
+            // anything faster is a window scroll
+            if (performance.now() - touchStartTime < panOnWaitDelay && intent == '') {
+              intent = INTENT_SCROLL;
+              setDebugBorder('1px solid red');
+            }
+            if (intent == '') {
+              setDebugBorder('1px solid green');
+              intent = INTENT_PAN;
+            }
+          }
+          // if we are ignoring a single finger touch, or it's a window-scroll, just 'return'
+          if ((intent == '' && ignoreSingleFingerTouch == true) || intent == INTENT_SCROLL) {
+            // have CanvasPanel do nothing... scroll the page
+            return;
+          }
```

Comment:
A nice interaction pattern 👍

Reply:
I want to run it through some user testing, but this felt right